pkg/api/common/versions.go

```diff
@@ -75,6 +75,7 @@ var AllKubernetesSupportedVersions = map[string]bool{
 	"1.11.0":         false,
 	"1.11.1":         true,
 	"1.11.2":         true,
+	"1.11.3":         true,
 	"1.12.0-alpha.1": true,
```

Comment:
Should we set 1.11.1 support to false since it is no longer one of the two latest patches?

Reply:
Yes we should--I'll add that.